src/test/java/bio/terra/cda/app/controller/QueryApiControllerTest.java

```diff
@@ -57,4 +57,25 @@ void booleanQueryDryRun() throws Exception {
     callQueryApi(false);
     verify(queryService, only()).startQuery(anyString());
   }
+
+  @Test
+  public void uniqueValuesTest() throws Exception {
+    String version = "v3";
+    String system = "GDC";
+    String body = "sex";
+    var expected =
+        "SELECT DISTINCT sex FROM TABLE.v3, UNNEST(ResearchSubject) AS _ResearchSubject, UNNEST(_ResearchSubject.identifier) AS _identifier WHERE _identifier.system = 'GDC' ORDER BY sex";
+    var result =
+        mvc.perform(
+                post("/api/v1/unique-values/v3")
```

Comment:
I think you can use templating here
```suggestion
                post("/api/v1/unique-values/version={version}", version)
```